server/api/cvrs.py

```diff
@@ -1050,29 +1082,14 @@ def parse_interpretations(cvr_xml: ET.ElementTree):
 
         return ",".join(interpretations)
 
-    # When only one CVR zip file is provided, we use just the batch name as the unique identifier
-    # for a batch and expect batch names to be unique across tabulators.
-    # Otherwise, when more than one CVR zip file is provided, we use the zip file names as
-    # tabulator names, and batch names need not be unique.
-    # For the former, check that the batch names are actually unique before proceeding.
     duplicate_batch_name = find_first_duplicate(
         batch.name for batch in jurisdiction.batches
     )
```

Comment:
Given how this variable is now used, might be clearer to do something like:
```py
use_tabulator_in_batch_key = find_first_duplicate(...) is not None
```

Reply:
Nice, I like that! https://github.com/votingworks/arlo/pull/1869/commits/869d91c6725ad821898b37f5aae74be8b3224d15